src/clipboard.js

```diff
@@ -1,10 +1,9 @@
 import $ from 'jquery'
 import loadingImg from './loadingImg'
 import * as u from './util'
-import * as Bacon from 'baconjs'
 import { invalidImageSelector } from './util'
 
-const SCREENSHOT_LIMIT_ERROR = () => new Bacon.Error('Screenshot limit reached!')
```

Comment:
Tätä pitäis ehkä vähän vielä funtsia. render-exam olettaa tällä hetkellä, että onValueChanged [kelpaa suoraan `Bacon.fromBinderille`](https://github.com/digabi/render-exam/blob/master/src/render-exam.js#L209). Eli virheet pitäisi kääriä `Bacon.Erroriin`. 

Exam-engine taasen olettaa tällä hetkellä, että [virhe-oliosta löytyy `error`-niminen property](https://github.com/digabi/exam-engine/blob/master/packages/core/src/components/RichTextAnswer.tsx#L82).

Reply:
tosta imho vois tehdä rikkovan muutoksen ja poistaa ton fromBinder-vaatimuksen